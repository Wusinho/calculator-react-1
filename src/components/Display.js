import React from 'react';
import PropTypes from 'prop-types';

const Display = (props) => {
  const { result } = props;
  return <h5>{result}</h5>;
};

Display.propTypes = {
  result: PropTypes.string,
};

Display.defaultProps = {
  result: '0',
};

export default Display;
