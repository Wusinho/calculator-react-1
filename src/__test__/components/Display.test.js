import React from 'react';
import { create } from 'react-test-renderer';
import Display from '../../components/Display';

it('Should render correctly', () => {
  const elem = create(<Display result="Some value" />);
  expect(elem).toMatchSnapshot();
});
