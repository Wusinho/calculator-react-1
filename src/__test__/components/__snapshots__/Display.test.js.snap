// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render correctly 1`] = `
<div
  className="display"
>
  <h5
    className="resultDisplayer"
  >
    Some value
  </h5>
  <h5
    className="partialDisplayer"
  >
    0 0
  </h5>
</div>
`;
